ios/RN/BarcodeDetectorManagerMlkit.m

```diff
@@ -242,6 +245,54 @@ - (NSArray *)processBarcodes:(NSArray *)barcodes imageContainingBarcodes:(UIImag
     return result;
 }
 
+- (NSString *)getFormat:(long)type
+{
+    NSString *barcodeType = @"UNKNOWN";
```

Comment:
I don't really know about objective-c, but this seems like a pretty low risk change.